pages/linux/screenkey.md

```diff
@@ -0,0 +1,32 @@
+# screenkey
+
+> A screencast tool to display keys pressed.
+> More information: <https://www.thregr.org/~wavexx/software/screenkey/>.
+
+- Display keys which are currently being pressed on the screen:
+
+`screenkey`
+
+- Display keys and mouse buttons which are currently being pressed on the screen:
+
+`screenkey --mouse`
+
+- Launch the settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Launch screenkey at a specific position:
+
+`screenkey --position {{top|center|bottom|fixed}}`
+
+- Change the format of the key modifiers displayed on screen:
+
+`screenkey --mods-mode {{normal|emacs|mac|win|tux}}`
+
+- Change the appearance of screenkey:
+
+`screenkey --bg-color "{{#353535}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
+
+- Drag and select a window on screen to display screenkey:
+
+`screenkey --position fixed --geometry {{$(slop -n -f '%g')}}`
```

Comment:
> screenkey: error: argument -g/--geometry: invalid geometry value: '{{330x157+1314+533}}'

Doesn't really work for me. I can select an area, but then I get the above error.

Reply:
The double curly brackets `{{}}` should not be included when the command is run in the terminal. It is a token syntax to show user provided values which tldr clients will need to highlight it.

Reply:
Being a tldr maintainer for a few months and still forgetting that `{{ }}` is tldr-only syntax... 😄